guide/using-the-e-puck-robot.md

```diff
@@ -40,29 +48,36 @@ E-puck is equipped with a large number of devices, as summarized in [this table]
 | Remote Control | infra-red LED for receiving standard remote control commands                                                                     |
 | Expansion bus  | expansion bus to add new possibilities to your robot                                                                             |
 | Programming    | C programming with the GNU GCC compiler system                                                                                   |
-| Simulation     | Webots EDU or PRO facilitates the programming of e-puck with a powerful simulation, remote control and cross-compilation system. |
+| Simulation     | Webots EDU or PRO facilitates the programming of e-puck with a powerful simulation, remote control and cross-compilation system  |
 
 %end
 
-### Simulation model
+### E-puck model
 
-%figure "The e-puck model in Webots"
+%figure "E-puck characteristics"
 
-![e-puck.png](images/e-puck.png)
+| Characteristics             | Values       |
+| --------------------------- | ------------ |
+| Diameter                    | 71 mm        |
+| Height                      | 50 mm        |
+| Wheel radius                | 20.5 mm      |
+| Axle Length                 | 52 mm        |
+| Weight                      | 0.16 kg      |
+| Max. forward/backward speed | 0.25 m/s     |
+| Max. rotation speed         | 6.28 rad/s   |
 
 %end
 
-The e-puck model in Webots is depicted in [this figure](#the-e-puck-model-in-webots).
-This model includes support for the differential wheel motors (encoders are also simulated, as position sensors), the infra-red sensors for proximity and light measurements, the accelerometer, the camera, the 8 surrounding LEDs, the body and front LEDs; the other e-puck devices are not yet simulated in the current model.
-Besides standard e-puck devices, the model also provides two slots that can be used to extend the functionality and to equip the robot with additional devices.
-In particular, the ground sensors module extension of the real e-puck robot is modeled in Webots to provide 3 optional infra-red sensors pointing to the ground in front of the robot.
-The [table](#devices-names) displays the names of the simulated devices which are to be used as an argument of the `wb_robot_get_device` function (see the Robot section of the [Reference Manual](http://www.cyberbotics.com/reference/)).
+The standard model of the e-puck is provided in the "E-puck.proto" PROTO file which is located in the "WEBOTS\_HOME/projects/robots/e-puck/protos" directory of the Webots distribution (see also "E-puckDistanceSensor.proto" PROTO file and "E-puckGroundSensors.proto" PROTO file); you will find complete specifications in it.
+The two PROTO fields `groundSensorSlot` and `turretSlot` have been included in the simulation model in order to support extension modules.
+In particular, the ground sensors module extension of the real e-puck robot is modelled in Webots to provide 3 optional infra-red sensors pointing to the ground in front of the robot.
```

Comment:
You could add here that the gound sensor extension is modeled using the `E-puckGroundSensors` PROTO, therefore, adding this PROTO into the  `groundSensorSlot` slot is doing the job. (but try to formulate this better :-) )